test/testdata/rewriter/t_enum_snapshot.rb.cfg.exp

```diff
@@ -46,7 +46,7 @@ subgraph "cluster_::<Class:MyEnum>#<static-init>" {
 T.class_of(Sorbet::Private::Static).keep_for_typechecking(<cfgAlias>$40: T.class_of(MyEnum::X))\l<cfgAlias>$43: T.class_of(MyEnum::X) = alias <C X$1>\l<castTemp>$41: MyEnum::X = <cfgAlias>$43: T.class_of(MyEnum::X).new()\l<C X>$35: MyEnum::X = <castTemp>$41\l<cfgAlias>$47: T.class_of(<Magic>) = alias <C <Magic>>\l<cfgAlias>$49: T.class_of(MyEnum::Y) = alias <C Y$1>\l<statTemp>$45: Sorbet::Private::Static::Void = <cfgAlias>$47: T.class_of(<Magic>).<define-top-class-or-module>(<cfgAlias>$49: T.class_of(MyEnum::Y))\l<cfgAlias>$52: T.class_of(Sorbet::Private::Static) = alias <C Static>\l<cfgAlias>$54: T.class_of(MyEnum::Y) = alias <C Y$1>\l<statTemp>$50: Sorbet::Private::Static::Void = <cfgAlias>$52: T.class_of(Sorbet::Private::Static).keep_for_ide(<cfgAlias>$54: T.class_of(MyEnum::Y))\l<cfgAlias>$57: T.class_of(Sorbet::Private::Static) = alias <C Static>\l<cfgAlias>$59: T.class_of(MyEnum) = alias <C MyEnum>\l<statTemp>$55: Sorbet::Private::Static::Void = <cfgAlias>$57: T.class_of(Sorbet::Private::Static).keep_for_ide(<cfgAlias>$59: T.class_of(MyEnum))\l<cfgAlias>$64: T.class_of(Sorbet::Private::Static) = alias <C Static>\l<cfgAlias>$66: T.class_of(MyEnum::Y) = alias <C Y$1>\l<statTemp>$62: Sorbet::Private::Static::Void = <cfgAlias>$64: T.class_of(Sorbet::Private::Static).keep_for_typechecking(<cfgAlias>$66: T.class_of(MyEnum::Y))\l<cfgAlias>$69: T.class_of(MyEnum::Y) = alias <C Y$1>\l<statTemp>$70: String(\"y\") = \"y\"\l<castTemp>$67: MyEnum::Y = <cfgAlias>$69: T.class_of(MyEnum::Y).new(<statTemp>$70: String(\"y\"))\l<C Y>$61: MyEnum::Y = <castTemp>$67\l<cfgAlias>$74: T.class_of(<Magic>) = alias <C <Magic>>\l<cfgAlias>$76: T.class_of(MyEnum::Z) = alias <C Z$1>\l<statTemp>$72: Sorbet::Private::Static::Void = <cfgAlias>$74: T.class_of(<Magic>).<define-top-class-or-module>(<cfgAlias>$76: T.class_of(MyEnum::Z))\l<cfgAlias>$79: T.class_of(Sorbet::Private::Static) = alias <C Static>\l<cfgAlias>$81: T.class_of(MyEnum::Z) = alias <C Z$1>\l<statTemp>$77: Sorbet::Private::Static::Void = <cfgAlias>$79: T.class_of(Sorbet::Private::Static).keep_for_ide(<cfgAlias>$81: T.class_of(MyEnum::Z))\l<cfgAlias>$84: T.class_of(Sorbet::Private::Static) = alias <C Static>\l<cfgAlias>$86: T.class_of(MyEnum) = alias <C MyEnum>\l<statTemp>$82: Sorbet::Private::Static::Void = <cfgAlias>$84: T.class_of(Sorbet::Private::Static).keep_for_ide(<cfgAlias>$86: T.class_of(MyEnum))\l<cfgAlias>$91: T.class_of(Sorbet::Private::Static) = alias <C Static>\l<cfgAlias>$93: T.class_of(MyEnum::Z) = alias <C Z$1>\l<statTemp>$89: Sorbet::Private::Static::Void = <cfgAlias>$91: T.class_of(Sorbet::Private::Static).keep_for_typechecking(<cfgAlias>$93: T.class_of(MyEnum::Z))\l<cfgAlias>$96: T.class_of(MyEnum::Z) = alias <C Z$1>\l<castTemp>$94: MyEnum::Z = <cfgAlias>$96: T.class_of(MyEnum::Z).new(<self>: T.class_of(MyEnum))\l<C Z>$88: MyEnum::Z = <castTemp>$94\l<blockReturnTemp>$17: NilClass = nil\l<blockReturnTemp>$98: T.noreturn = blockreturn<enums> <blockReturnTemp>$17: NilClass\l<unconditional>\l"
```

Comment:
this diff looks ugly so here's a slightly better `git diff --word-diff`

![image](https://user-images.githubusercontent.com/5544532/85622072-21766180-b61b-11ea-8228-b6152513310b.png)